README.md

```diff
@@ -86,7 +86,8 @@ This project uses Docker and Docker Compose to run the application, which includ
 - Start all services defined in the docker-compose.yml file (e.g., frontend, backend, database).
 
 2.  **Access the Application**:
-    - The app is running at http://localhost:3000.
+
+    - The app is running at http://localhost:3000. Note that this may conflict with your local dev server. If so, one will be running on port 3000 and the other on port 3001.
```

Comment:
added a note here based on our convo regarding local dev vs Docker front end @agennadi; it could probably be clarified further, but what do you think?

Reply:
cc @mantuok would like your feedback as well since you'll have a good read on how a new front end dev would read this

Reply:
Thank you for adding this note, @nickvisut.  I am going to update the README and mention that there are two separate options for running the app: locally and in Docker.